dev-docs/howto/vpn/helm/README.md

```diff
@@ -2,35 +2,81 @@
 
 This Helm chart deploys a VPN server to your Constellation cluster.
 
-## Installation
+## Prerequisites
 
-1. Create and populate the configuration.
+* Constellation >= v2.14.0
+* A publicly routable VPN endpoint on premises that supports any of
+  * IPSec in IKEv2 tunnel mode with NAT traversal enabled
+  * Wireguard
+* A list of on-prem CIDRs that should be reachable from Constellation.
+
+## Setup
+
+1. Configure Constellation's Cilium for the VPN.
+   * Add all on-prem CIDRs to the `nonMasqueradeCIDRs` in the `ip-masq-agent` ConfigMap. See
+     <https://docs.cilium.io/en/stable/network/concepts/masquerading/#ebpf-based>
+     for more background information.
+   * TODO: explain how to enable `enable-sctp`
```

Comment:
1. Putting it in the operator sounds about right, but I'd not create the config if its absent (see below).
2. We could discuss whether Constellation should be touching cm/ip-masq-agent at all, because I think the defaults would work just fine for us and by owning it we make it unavailable for cluster admins.
3. I would like to avoid going back to the tproxy.

Reply:
Moved into the operator.

Reply:
This comment was regarding line 19 and `enable-sctp`. This is not part of the operator is it?

Reply:
I don't think the operator should be modifying the Cilium config. I also don't think it's great that the operator modifies the ip-masq-agent config, but I think I'd draw the line at Cilium: just too worried that the operator might break stuff.